README.md

```diff
@@ -6,20 +6,28 @@ Created and Open Sourced thanks to [LocalBini.com](http://Localbini.com) by [pdy
 
 ![Python 3.8+](https://img.shields.io/badge/python-v3.8-blue) ![Black](https://img.shields.io/badge/code%20style-black-000000.svg)
 
-## Settings:
-`CLIENT_SECRET` For decoding authorization jwt. Defaults 'secret'.\
-`COGNITO_PUBLIC_KEYS` For validating IdToken signature when Cognito authentication enabled.\
-(simply paste from https://cognito-idp.{your aws region}.amazonaws.com/{your pool id}/.well-known/jwks.json) \
-`COGNITO_ALLOWED_CLIENTS` Allowed Cognito clients id's. String in format "id1,id2,id3". Used for verifying IdToken signature.\
-`PRINT_TRACEBACK` More verbose errors. Defaults False (set as "0" or "1").\
-`LOGGING_LEVEL` For logging level. Defaults INFO.\
-`EXPIRATION_KEY` for defining the expiration key used in authorization default "exp" according to JWT standard.
-`ALLOWED_ISS` for defining the allowed issuer of JWT - Security feature.
+## Configuration
+
+Lambdalizator can be configured using the following environment variables: 
+- `INTERNAL_AUTH_JWK` - a JSON Web Key that can be used for encoding and decoding internal JWTs.
+  It needs to be a JSON object that can be generated for example using [mkjwk](https://mkjwk.org/).
+- `ALLOWED_PUBLIC_KEYS` - a list of public keys that can be used for decoding auth tokens send in the
+  `Authentication` header. If you are using Cognito, you can use public keys from:
+  https://cognito-idp.{your aws region}.amazonaws.com/{your pool id}/.well-known/jwks.json.
+- `ALLOWED_AUDIENCES` - a list of audiences that will be used for verifying the JWTs send in the
+  `Authentication` header. It should be a comma-separated list of strings, e.g. `aud1,aud2`.
+  If not set, any audience will be considered valid.
+- `ALLOWED_ISS` - allowed issuer of JWT - Security feature. If not set, issuer will not be checked.
+- `AUTH_REMOVE_PREFIXES` - if enabled, all fields starting with a prefix (like `cognito:`) in the
```

Comment:
I think we should point out it's relevant for parsing JWT to an User instance - ultimately parsing raw JWT is not our goal - user instance does

Reply:
Same situation, I'd do an overhaul of the docs separately